.github/workflows/nightly-release.yml

```diff
@@ -62,17 +62,21 @@ jobs:
       - name: 🏷 Push Tag
         run: git push origin --tags
 
-      - name: 🔐 Setup npm auth
-        run: |
-          echo "registry=https://registry.npmjs.org" >> ~/.npmrc
-          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" >> ~/.npmrc
       - name: 🚀 Publish
-        run: npm run publish
+        run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: 🐱 Create GitHub release
```

Comment:
I think this will trigger the workflow in npm-publish.yml. Since it's a nightly release, we don't need to create a GitHub release for it.